hack/validate/vendor

```diff
@@ -3,10 +3,15 @@
 SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source "${SCRIPTDIR}/.validate"
 
+go_mod_tidy(){
+	${SCRIPTDIR}/../go-mod-prepare.sh
```

Comment:
Looks like this is missing some quotes, and the file itself doesn't appear to be included in this PR?